steps/oci-compute-step-images-describe/spec.schema.json

```diff
@@ -0,0 +1,57 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"type": "object",
+	"properties": {
+		"aws": {
```

Comment:
Should be `"oci"` here, and elsewhere in the schemas (same mistake is duplicated to all of them).